mediator/med_phases_aofluxes_mod.F90

```diff
@@ -429,17 +430,29 @@ subroutine med_aofluxes_run(gcomp, aoflux, rc)
     !----------------------------------
 
     if (first_call) then
-       call NUOPC_CompAttributeGet(gcomp, name='coldair_outbreak_mod', value=cvalue, rc=rc)
+       call NUOPC_CompAttributeGet(gcomp, name='coldair_outbreak_mod', value=cvalue, isPresent=isPresent, isSet=isSet, rc=rc)
        if (chkerr(rc,__LINE__,u_FILE_u)) return
-       read(cvalue,*) coldair_outbreak_mod
+       if (isPresent .and. isSet) then
+          read(cvalue,*) coldair_outbreak_mod
+       else
+          coldair_outbreak_mod = .false.
+       end if
 
-       call NUOPC_CompAttributeGet(gcomp, name='flux_max_iteration', value=cvalue, rc=rc)
+       call NUOPC_CompAttributeGet(gcomp, name='flux_max_iteration', value=cvalue, isPresent=isPresent, isSet=isSet, rc=rc)
        if (chkerr(rc,__LINE__,u_FILE_u)) return
-       read(cvalue,*) flux_max_iteration
+       if (isPresent .and. isSet) then
+          read(cvalue,*) flux_max_iteration
+       else
+          flux_max_iteration = 1
+       end if
 
-       call NUOPC_CompAttributeGet(gcomp, name='flux_convergence', value=cvalue, rc=rc)
+       call NUOPC_CompAttributeGet(gcomp, name='flux_convergence', value=cvalue, isPresent=isPresent, isSet=isSet, rc=rc)
        if (chkerr(rc,__LINE__,u_FILE_u)) return
-       read(cvalue,*) flux_convergence
+       if (isPresent .and. isSet) then
+          read(cvalue,*) flux_convergence
```

Comment:
The default value of flux_convergence is 0.01 in namelist_definition_drv.xml, I think it is better to set it 0.01 if it is not set.